components/nimbus/src/schema.rs

```diff
@@ -4,7 +4,7 @@
 
 use crate::{defaults::Defaults, enrollment::ExperimentMetadata, NimbusError, Result};
 use serde_derive::*;
-use serde_json::{Map, Value};
+use serde_json::{from_str, from_value, Map, Value};
```

Comment:
Nit: I'd prefer to see well known functions like this be used explicitly/qualified.

i.e. use

```rust
let foo: Value = serde_json::from_str(payload)?;
```

rather than

```rust
let foo: Value = from_str(payload)?;
```

Reply:
Also: `serde_json::to_string`, `serde_json::to_value` etc.

Reply:
Good to know — totally fine with that change. Out of curiosity, is there a particular reason for explicit qualification vs `use`ing?